modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -2,13 +2,8 @@
 
 @import 'management-toolbar';
 
-.side-panel-iframe__tabs {
-	background-color: $white;
-	padding: 0.5rem 1rem 0;
-
-	.nav-link {
-		padding: 1rem;
-	}
+.nav-tabs + .tab-content .tab-pane {
+	padding: 1.5rem;
```

Comment:
Hi @Artartur. I've tested the PR and I noticed the following requirement ([ticket](https://issues.liferay.com/browse/LPS-186200)) was not being applied correctly for all the panels.

> The padding between the card and the sidebar should be 24px.

I talked to Julia and she said we need to get the same look of Field's sidepanel that is on master. So I made some tests now and I think the correct padding should be `0rem`.

![image](https://github.com/liferay-objects/liferay-portal/assets/26491796/397e1dcd-f09e-4cdd-84a0-75f5430902cd)

Please, could you test if this fixes all the panels?
```
.lfr-objects__side-panel-content {
	.nav-tabs + .tab-content .tab-pane {
		padding: 0rem;
	}
}
```

Thanks! :relaxed:

Reply:
Yeah you're right, I've tested here your request with the code you sent and it fixed all the panels. Thx @alinedoleron

Reply:
Thank you for the quick response and update, @Artartur! ☺️

Reply:
My bad, I fixed the SF problem